app/src/main/kotlin/com/simplemobiletools/calculator/activities/BitwiseActivity.kt

```diff
@@ -0,0 +1,167 @@
+package com.simplemobiletools.calculator.activities
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.view.View
+import com.simplemobiletools.calculator.R
+import com.simplemobiletools.calculator.helpers.BitwiseCalculatorImpl
+import com.simpletools.calculator.commons.activities.SimpleActivity
+import com.simpletools.calculator.commons.extensions.config
+import com.simpletools.calculator.commons.extensions.updateViewColors
+
+/* ktlint-disable no-wildcard-imports */
+import com.simpletools.calculator.commons.helpers.*
+import kotlinx.android.synthetic.main.activity_base.*
+import com.simplemobiletools.commons.extensions.*
+/* ktlint-enable no-wildcard-imports */
+
+class BitwiseActivity : SimpleActivity(), Calculator {
```

Comment:
Glad the name was changed clears up alot of confusion!

Reply:
Indeed